tests/unit/acquisition/function/test_function.py

```diff
@@ -704,3 +706,29 @@ def test_multiple_optimism_negative_confidence_bound_raises_for_changing_batch_s
     with pytest.raises(tf.errors.InvalidArgumentError):
         query_at = tf.reshape(tf.linspace([[-10]], [[10]], 100), [5, 10, 2])
         acq_fn(query_at)
+
+
+@pytest.mark.parametrize("in_place_update", [False, True])
```

Comment:
whats this in place about?

Reply:
This checks that MakePositive's update method works both when the underlying update method updates the acquisition function "in place" (when we don't have to regenerate the wrapper) and when the underlying update method creates a new acquisition function (when we do).